netci.groovy

```diff
@@ -210,6 +210,11 @@ class Constants {
                 'Checked'
             ]
         ],
+        'Tizen': [
+            'armem': [
```

Comment:
@gbalykov should this be "armel" instead of "armem"?

Reply:
@janvorli, this is simply revert of part of #17240. `armem` is used for ARM builds/runs using emulator.

Reply:
Ah, ok, thank you for the clarification!